### ISSUE TITLE
Poor accuracy and slow training when increasing satellite image size (DOTA)

### ISSUE THREAD

**Issue by matt-sharp** [AlexeyAB/darknet#7384]
I'm using the DOTA dataset which contains large satellite images and I have patched them into sub-images of 1024 x 1024 in size. 

https://arxiv.org/pdf/1711.10398.pdf

There are 15k training images after patching with 15 annotated classes of varying size, orientation and aspect ratio.
I have previously trained a model and achieved a mAP of 64% with 64k iterations after 3.5 days.  This was using a network size of 608 x 608.  I decided to increase the network size to 1024 x 1024 and kept everything else the same.  The reason is because I expect a larger network size to be better at identifying small objects.
The mAP is now stuck at 40% after 22,600 iterations which has taken 4 days to train.  I'm using 1 x Tesla v100.
I understand that training will be slower with a larger network size but should it be this slow?

My questions are as follows:

**1. Why is the mAP much lower with a larger image size?  Is this simply because it's taking longer to train?
2. What is the best strategy for improving performance with this type of data?
3. How can I speed up training?**

Here's the command I used:

./darknet detector train /jupyter/work/msharp/DOTA-yolov3/cfg/dota.data /jupyter/work/msharp/darknet/cfg/exp3_yolov4.cfg -dont_show -gpus 0 -map

```
 CUDA-version: 10010 (11000), cuDNN: 7.6.5, CUDNN_HALF=1, GPU count: 1
 CUDNN_HALF=1
 OpenCV version: 3.2.0
 0 : compute_capability = 700, cudnn_half = 1, GPU: Tesla V100-SXM2-16GB
   layer   filters  size/strd(dil)      input                output
   0 conv     32       3 x 3/ 1   1024 x1024 x   3 -> 1024 x1024 x  32 1.812 BF
```

Here's the top part of my config file:

[net]
# Testing
#batch=1
#subdivisions=1
# Training
batch=64
subdivisions=64
width=1024
height=1024
channels=3
momentum=0.949
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1

learning_rate=0.001
burn_in=1000
max_batches = 500200
policy=steps
steps=400000,450000
scales=.1,.1

#cutmix=1
mosaic=1

Here's the bottom part of my config file:

[convolutional]
size=1
stride=1
pad=1
filters=60
activation=linear

[yolo]
mask = 6,7,8
anchors = 12, 16, 19, 36, 40, 28, 36, 75, 76, 55, 72, 146, 142, 110, 192, 243, 459, 401
classes=15
num=9
jitter=.3
ignore_thresh = .7
truth_thresh = 1
random=1
scale_x_y = 1.05
iou_thresh=0.213
cls_normalizer=1.0
iou_normalizer=0.07
iou_loss=ciou
nms_kind=greedynms
beta_nms=0.6
max_delta=5

![image](https://user-images.githubusercontent.com/13571774/108057035-fa591e00-7049-11eb-8619-b8f114f0a29b.png)

**Comment by Taotaoxu** [comment]
Did you check your label of patches? 
I also tried the [YOLOv4-CSP-Version](https://github.com/WongKinYiu/ScaledYOLOv4/tree/yolov4-csp) under the PyTorch framework with 1024\*1024 patches on the **DOTA** train set and got an overall testing mAP of 70.57% (from the DOTA webSite) without any trick (such as Multi-scale training or testing). 
My input size is 512\*512, and the anchor size is the default setting, furthermore, no more setting in yolov4-csp.cfg file is changed.
